addons/l10n_be_edi/models/account_invoice.py

```diff
@@ -135,7 +135,10 @@ def _decode_ubl_2_1(self, tree):
 
                     # Price Unit
                     elements = eline.xpath('cac:Price/cbc:PriceAmount', namespaces=namespaces)
-                    invoice_line_form.price_unit = elements and float(elements[0].text) or 0.0
+                    price_unit = elements and float(elements[0].text) or 0.0
+                    elements = eline.xpath('cbc:LineExtensionAmount', namespaces=namespaces)
+                    line_extension_amount = elements and float(elements[0].text) or 0.0
+                    invoice_line_form.price_unit = price_unit or line_extension_amount / invoice_line_form.quantity or 0.0
```

Comment:
Can this be zero? `invoice_line_form.quantity`

Reply:
looking at the line above
```quantity = elements and float(elements[0].text) or 1.0```
I guess no

Edit: I accidentally started a review on a review, instead of just commenting